src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -290,11 +301,18 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 	}
 
+	@debounce(50)
 	private scrollToActiveCell(): void {
 		// Get active cell from active notebook editor
-		const activeCellElement = document.querySelector(`.editor-group-container.active .notebook-cell.active`);
-		activeCellElement.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+		const activeCellElement = this.container.nativeElement.querySelector(`.editor-group-container.active .notebook-cell.active`);
```

Comment:
While you're here - this shouldn't be necessary to run again. Just pass in the active cell since we already got it in the event handler above.

Reply:
We need to get the active cell again because the active cell that we got in the event handler is the past cell (before selecting the next cell)